webhook_server/app.py

```
@@ -1127,6 +1260,850 @@ async def websocket_log_stream(
+    if start_datetime:
+        query += f" AND created_at >= ${param_idx}"
+        params.append(start_datetime)
+        param_idx += 1
+
+    if end_datetime:
+        query += f" AND created_at <= ${param_idx}"
+        params.append(end_datetime)
+        param_idx += 1
+
+    # Get total count for pagination
+    count_query = f"SELECT COUNT(*) FROM ({query}) AS filtered"
```

Ruff (0.14.5)

Possible SQL injection vector through string-based query construction

(S608)